pkg/operator/starter.go

```diff
@@ -242,6 +252,7 @@ var RevisionSecrets = []revision.RevisionResource{
 
 var CertConfigMaps = []revision.RevisionResource{
 	{Name: "restore-etcd-pod"},
+	{Name: "etcd-scripts"},
```

Comment:
I called them "recovery-scripts". This is already etcd pod, so ...

Reply:
> I called them "recovery-scripts". This is already etcd pod, so ...

I think these may contain more than recovery scripts.